README.md

```diff
@@ -175,15 +175,19 @@ DICOM format rules are much the same:
 dicom:
     associated_images:
         ...
-    delete_custom_metadata: ...
+    custom_metadata_action: ...
     metadata:
         ...
 ```
 
-Note that here there is an eplicit format-level setting for dealing with custom metadata. Any tag with an odd group number is custom metadata. If not specified, this value default to `True`. Tag-level rules will override this behavior.
+Note that here there is an eplicit format-level setting for dealing with custom metadata. Any tag with an odd group number is considered custom metadata. This can be set to `keep`, `delete` or `use_rule`.
+
+* `keep`: Retain the custom metadata value after redaction.
+* `delete`: Delete the custom metadata tag from the image.
+* `use_rule`: This mode will fall back to rules specified for each piece of custom metadata in the `metadata` section of the rule set. If a custom metadata tag with no corresponding rule is encountered, the image will not be redacted, as the redaction plan would be considered incomplete.
```

Comment:
Tag-level rules always apply even with `keep` or `delete`, right?  If so, we should mention that, too.